### ISSUE TITLE
XCode7 bitcode

### ISSUE THREAD

**Issue by drico** [keenlabs/KeenClient-iOS#122]
When compiling on a device with XCode 7, I get the following issue.
```
ld: '/Users/*****/KeenClient/libKeenClient-Aggregate.a(HTTPCodes.o)' 
does not contain bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), 
obtain an updated library from the vendor, or disable bitcode for this target. 
for architecture armv7
```
Of course, setting `ENABLE_BITCODE` to `NO` will solve the issue in an app, but I am developping an SDK that embeds KeenIO SDK.

Thus, to make my SDK BITCODE ready, I need the KeenIO SDK to be BITCODE ready too.

**Comment by heitortsergent** [comment]
@drico sorry for the delay in getting back to you on this.

Could you please see if version 3.5.0 of the SDK fixes the issue for you? I just uploaded it [here](https://github.com/keenlabs/KeenClient-iOS/releases), and it seems to be working on my end. I tested with version 3.4.3 before and it really wasn't working.

**Comment by drico** [comment]
Hi, I confirm this is fixed in the 3.5.0 release :)
Thank you !

**Comment by heitortsergent** [comment]
Awesome, thanks for confirming it @drico :100:
